packages/cdk8s-cli/lib/import/base.ts

```diff
@@ -22,15 +23,15 @@ export abstract class ImportBase {
   public abstract get moduleNames(): string[];
   protected abstract async generateTypeScript(code: CodeMaker, moduleName?: string): Promise<void>;
 
-  public async import(options: ImportOptions) {
+  public async import(options: ImportOptions, source?: Cdk8sImport) {
```

Comment:
The `spec` should go in the constructor instead of `moduleName` there.

Reply:
Sorry, I'm not following this one. What do you mean here?

Reply:
I think this is resolved now.. `source` is now named `importSpec`